mitodl/unified-ecommerce
# 5654 update apis for shopping cartbasket

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/5654

### Description (What does it do?)
Allows an administrator to define a payment_process_redirect_url, in the Django Admin, which is where the user is redirected to when leaving CyberSource.  The payment_process_redirect_url is defined as part of the Integrated System object.
![Screenshot 2024-10-16 at 8 57 49 AM](https://github.com/user-attachments/assets/42901700-223d-4f92-88ef-e7df4346cb67)


### How can this be tested?

1. Run the `generate_test_data` management command.
2. Login to the Django Admin and add different URLs for the payment_process_redirect_url of the Integrated System objects.
3. Go to the mule application page in your browser.
4. Add two items, each associated with different Integrated Systems.  Make note of which is the second item you add to your cart.
5. Complete the checkout process.
6. Verify that you are redirected to the payment_process_redirect_url associated with the second item you added to your cart at step 4.
